en/frames_rover/index.md

```diff
@@ -28,6 +28,10 @@ The supported frames can be seen in [Airframes Reference > Rover](../airframes/a
 
 ### Ackermann Steering Configuration
 
+:::warning
```

Comment:
How would you characterise the old ackermann vs new module in "v1.16"? Will the old module still exist in v1.16, or will it be removed, or will it be deprecated?

How about right now if someone is working on main? What should they use?

The reason I ask is that if the "old" ackermann module is going to be gone then I would work towards removing it from the docs now. We'd structure as differential and ackermann. I can do this, but I need to know what the intent is and how "firm" it is.

Reply:
@chfriedrich98 This comment above relates to your comment:

> I think this PR could be merged on its own for now, since it is esentially a standalone file. I

I don't think it is standalone, though you're quite right we don't need to fully split everything up now. But if we know that this module is going to be the only thing we would recommend for Ackermann right now, we can very much restructure this to hide the old stuff.

If that is the case, I would also perhaps suggest removing the old airframe so people can't select it.

Reply:
> How would you characterise the old ackermann vs new module in "v1.16"? Will the old module still exist in v1.16, or will it be removed, or will it be deprecated?
How about right now if someone is working on main? What should they use?

The old module uses a guidance algorithm that was designed for fixed-wing and the module was used for all types of USV's but didn't do a great job with any of them. This is why we started creating dedicated modules for the different USV's. The feedback I have received so far is that the new module works significantly better for ackermann rovers than the old one, so I would definitely recommend using the new one.

> The reason I ask is that if the "old" ackermann module is going to be gone then I would work towards removing it from the docs now. We'd structure as differential and ackermann. I can do this, but I need to know what the intent is and how "firm" it is.

I want to remove the old module once the differential and boat modules are properly done, but there might be some pushback on that, so I am not sure yet.

> I don't think it is standalone - it is confusing as is. Though you're quite right we don't need to fully split everything up now. But if we know that this module is going to be the only thing we would recommend for Ackermann right now, we can very much restructure this to hide the old stuff.

I agree it is quite confusing like this, hiding the old stuff sounds like a good idea.

> If that is the case, I would also perhaps suggest removing the old airframe so people can't select it. You could even just name it with "deprecated" in the title.

Good idea, I'll look into that!